plugins/auth-backend/src/lib/oauth/helpers.test.ts

```diff
@@ -24,20 +24,26 @@ describe('OAuthProvider Utils', () => {
         nonce: '123',
         env: 'development',
         origin: 'https://example.com',
+        extraValue: 'somevalue',
       };
 
       const encoded = encodeState(state);
       expect(encoded).toBe(
         Buffer.from(
-          'nonce=123&env=development&origin=https%3A%2F%2Fexample.com',
+          'nonce=123&env=development&origin=https%3A%2F%2Fexample.com&extraValue=somevalue',
         ).toString('hex'),
       );
 
       expect(readState(encoded)).toEqual(state);
     });
 
     it('should not include undefined values', () => {
-      const state = { nonce: '123', env: 'development', origin: undefined };
+      const state = {
+        nonce: '123',
+        env: 'development',
+        origin: undefined,
+        extraValue: undefined,
```

Comment:
should this be extraState?

Reply:
Fuck yes. Ugh thanks you

Reply:
Well actually, it doesn't really matter in this case but yes. I'll do it for continuity.